app/models/miq_request_workflow.rb

```diff
@@ -501,6 +501,16 @@ def required_description(dlg, fld)
     "'#{dlg[:description]}/#{fld[:required_description] || fld[:description]}'"
   end
 
+  def allowed_storage_profiles(options = {})
```

Comment:
If this method is only for VMware it should go in the VMware workflow: [app/models/manageiq/providers/vmware/infra_manager/provision_workflow.rb](https://github.com/ManageIQ/manageiq/blob/master/app/models/manageiq/providers/vmware/infra_manager/provision_workflow.rb)